### ISSUE TITLE
Add option to select a single KVR

### ISSUE THREAD

**Issue by thielepaul** [okainov/munich-scripts#76]
If one is living very close to one of the KVRs then it would be useful to limit the search to that one

**Comment by okainov** [comment]
Agree, might make sense (though I barely see the point as I consider the case when there are no available appointments at all and one would be happy to grab any), can be added probably as one extra step during search/booking
